Dockerfile

```diff
@@ -25,10 +25,6 @@ RUN set -ex; \
   \
   rm -rf /opt/kafka/site-docs; \
   \
-  mkdir -p /opt/kafka/prometheus; \
```

Comment:
why this change?

Reply:
the helm chart does not use `jmx_prometheus_javaagent.jar` in kafka container for exporting JMX metrics. It uses a sidecar container `prometheus-jmx-exporter` for this.